packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -29,115 +31,98 @@ keywords: [theme, locale, scale, toast, form props]
+### Color Schemes
+We recommend supporting both light and dark color schemes in your app, however, if you need to override this with an application specific setting, you can use the `colorScheme` prop.
 
 ```tsx example
-<Provider colorScheme="dark" scale="large">
-  <Form margin="15px">
-    <div>
-      <TextField label="Email" />
-    </div>
-    <Checkbox>
-      Checkbox Label
-    </Checkbox>
-    <div style={{'marginBottom': '15px'}}>
-      <Button variant="primary">Button</Button>
-    </div>
-  </Form>
+<Provider theme={theme} colorScheme="light">
+  <ActionButton>I'm a light button</ActionButton>
 </Provider>
 ```
 
-### Theme
+See the [styling docs](styling.html) for more information about using Spectrum color variables in your app to ensure it adapts to light and dark mode properly.
 
-The Provider's `theme` is the CSS variables for `colorScheme` and `scale` values.
+### Locales
+Another important setting for your application is the locale. By default, React Spectrum chooses the locale matching the user’s browser/operating system language, but this can be overridden with the locale prop if you have an application specific setting. This prop accepts a [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt) language code. A list of supported locales is available [here](https://react-spectrum.adobe.com/react-aria/internationalization.html#supported-locales).
 
-```tsx
-import {theme} from '@react-spectrum/theme-default';
 
-<Provider theme={theme} colorScheme="dark" scale="large">
-  <Form>
-    <div>
-      <TextField label="Email" />
-    </div>
-    <Checkbox>
-      Checkbox Label
-    </Checkbox>
-    <div>
-      <Button variant="primary">Button</Button>
-    </div>
-  </Form>
-</Provider>
-```
-
-### Locale
-
-The `locale` prop can be set to specify a locale for children of the Provider.
-The default for the locale prop is based on the system/browser settings.
-The `locale` prop does not support nesting. It must be the same across the page and all `lang`/`dir` attributes in the dom must match.
-See this list of supported [locales](../react-aria/internationalization#supported-locales).
-
-```tsx example
-<Provider locale="en-US">
-  <Form necessityIndicator="label">
-    <div>
-      <TextField label="email" />
-    </div>
-    <div>
-      <Button variant="primary">Button</Button>
-    </div>
-  </Form>
+```tsx
+<Provider theme={theme} locale={appSettings.locale}>
+  <YourApp />
 </Provider>
 ```
 
-## Provider nesting
+To access the current locale anywhere in your application, see the [useLocale](https://react-spectrum.adobe.com/react-aria/useLocale.html) hook.
 
-Providers can be nested. The parent Provider passes its props down to the
-children Providers.
+## Property groups
+Provider can also be used to define common properties for a group of components within in. For example, a group of components could be disabled together rather than individually by setting the `isDisabled` prop on any parent provider.
```

Comment:
```suggestion
Provider can also be used to define common properties for a group of components within it. For example, a group of components could be disabled together rather than individually by setting the `isDisabled` prop on any parent provider.
```